lib/config/presets/internal/group.ts

```diff
@@ -484,7 +484,7 @@ const staticGroups = {
     packageRules: [
       {
         groupName: 'spring boot',
-        matchPackageNames: ['org.springframework.boot'],
+        matchDepNames: ['org.springframework.boot'],
```

Comment:
The current PR titles is `fix: matchDepNames for sprintBoot`

I think you meant `springboot`, or `Spring Boot`?